presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PlanNodeSearcher.java

```diff
@@ -29,18 +30,32 @@
 
 public class PlanNodeSearcher
 {
+    @Deprecated
     public static PlanNodeSearcher searchFrom(PlanNode node)
     {
         return new PlanNodeSearcher(node);
     }
 
+    public static PlanNodeSearcher searchFrom(PlanNode node, Lookup lookup)
+    {
+        return new PlanNodeSearcher(node, lookup);
+    }
+
     private final PlanNode node;
+    private final Lookup lookup;
     private Predicate<PlanNode> where = alwaysTrue();
     private Predicate<PlanNode> skipOnly = alwaysTrue();
 
+    @Deprecated
     public PlanNodeSearcher(PlanNode node)
     {
+        this(node, (n) -> n);
```

Comment:
I can't. It's `Lookup` interface. Lambda here is just a shortcut to anonymous class implementation.